sonar-project.properties

```diff
@@ -11,3 +11,4 @@ sonar.exclusions=**/*.test.*
 sonar.coverage.exclusions=\
   **/vite.config.ts
 sonar.typescript.tsconfigPath=./tsconfig.json
+sonar.python.coverage.reportPaths=**/synapse/coverage.xml
```

Comment:
I dont think this accepts `**` wildcards, just set it to the one we have for now, we can tackle this problem later. The action we use has support for detecting all javascript covg files, we could extend it for python ones or we could just list them all here exhaustively comma-delimited.

Reply:
I'm happy to delegate this as long as the codecov failure isn't blocking the merge.  For now I'll revert this commit (but will leave in the ones that generate the coverage report)

Reply:
Right so I think the biggest issue here is that Sonarcloud is ran after the `Tests` workflow but the synapse code from this PR doesn't get its coverage generated there